docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,425 @@
+- Standard for [exporter components](https://opentelemetry.io/docs/collector/)
+- [OTLP wire protocol](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/README.md) for transmitting telemetry data
+
+There are OpenTelemetry implementations for most languages and platforms, including .NET.
+
+## .NET Observability architecture
+
+The .NET OpenTelemetry implementation is a little different from other platforms, as .NET provides logging, metrics, and activity APIs in the BCL. That means OTel doesn't need to provide APIs for library authors to use. The .NET implementation uses these existing APIs for instrumentation:
+
+- <xref:Microsoft.Extensions.Logging.ILogger%601?displayProperty=nameWithType> for [logging](../extensions/logging.md)
+- <xref:System.Diagnostics.Metrics.Meter?displayProperty=nameWithType> for [metrics](./metrics-instrumentation.md)
+- <xref:System.Diagnostics.ActivitySource?displayProperty=nameWithType> and
+<xref:System.Diagnostics.Activity?displayProperty=nameWithType> for [distributed tracing](./distributed-tracing.md)
+
+![.NET OTel architecture](./media/layered-approach.svg)
+
+Where OTel comes into play is that it collects telemetry from those APIs and other sources (via instrumentation libraries) and then exports them to an application performance monitoring (APM) system for storage and analysis. The benefit that OTel brings as an industry standard is a common mechanism for collection, common schemas and semantics for telemetry data, and an API for how APMs can integrate with OTel. Using OTel means that applications don't need to use APM-specific APIs or data structures; they work against the OTel standard. APMs can either implement an APM specific exporter component or use OTLP, which is a new wire standard for exporting telemetry data to the APM systems.
+
+Using OTel enables the use of a wide variety of APM systems including open-source systems such as [Prometheus](https://prometheus.io/) and [Graphana](https://grafana.com/oss/grafana/), [Azure Monitor](/azure/azure-monitor/app/app-insights-overview?tabs=net) - Microsoft's APM product in Azure, or from the many [APM vendors](https://opentelemetry.io/ecosystem/vendors/) that partner with OpenTelemetry.
+
+## OpenTelemetry packages
+
+OpenTelemetry in .NET is implemented as a series of Nuget packages that form a couple of categories:
+
+- Core API
+- Instrumentation providers - these collect instrumentation from the runtime and common libraries.
+- Exporters - these interface with APM systems such as Prometheus, Jaeger, and OTLP.
+
+The following table describes the main packages.
+
+| Package Name | Description |
+| --- |  ---|
+| [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md) | Main library that provides the core OTEL functionality |
+| [OpenTelemetry.API](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/README.md) | APIs for exporting metrics |
```

Comment:
The description "API for exporting Metrics" is incorrect.

Reply:
What is a better description?